Automattic/jetpack
# Remove save() function from subscription blocks

Fixes #26866.

#### Changes proposed in this Pull Request:
This patch fully removes the old `save()` implementation from the subscribe block, which has been rendered obsolete by #26947 where the block has been converted to be dynamic.

#### Other information:


#### Jetpack product discussion



N/A

#### Does this pull request change what data or activity we track or use?



No.

#### Testing instructions:






- Make sure Jetpack has a valid connection on your test site for subscribe blocks to be enabled.
- Create a new post and a subscribe block.
- Save.
- Preview the post, it should still feature the subscribe form you just added.
- Go back to the editor and switch the block to code editor.
- There should be no shortcode being generated for the block anymore.
